### DISCUSSION TITLE
Bull Workers

### DISCUSSION THREAD

**Discussion by LeonardoDB** [OptimalBits/bull#2400] [General]
Hello, how are you?

I have a question, can I limit the number of workers bull can create?

From my tests I noticed that it creates 3 workers per instance. I believe this could be an issue with the horizontal scaling of the application. For each instance created, three new workers will be generated.

**Comment by manast** [comment]
"3 workers per instance". Could you clarify what "instance" means in this context?

**Comment by LeonardoDB** [comment]
Instance of my application. I noticed that 3 new workers are generated for each bull process correct?

For example: In this case I created two instances of my application, one running on port 3001 and another on 3002, I told the 3001 application to display how many workers it has, there are 6.

![image](https://user-images.githubusercontent.com/33195732/179352874-3d05749c-4613-4b89-be98-9cf0249f332f.png)

**Comment by manast** [comment]
And how many workers do you have?

**Comment by LeonardoDB** [comment]
According to image 6, how can I define how many will be created? Does he manage for me?

**Comment by manast** [comment]
I don't know how your source code looks like...
